src/utils/track-utils.ts

```diff
@@ -212,18 +212,25 @@ export const HasMoveOperations = (tr: Transaction) => {
     }
     const step = tr.steps[i] as ReplaceStep
     const doc = tr.docs[i]
-    // const slice = step.slice.size === 0 ? doc.slice(step.from, step.to) : step.slice
+
+    if (!step.slice) {
+      // skipping step without slice
+      continue
+    }
+
     const stepDeletesContent = step.from !== step.to && step.slice.size === 0
     const stepInsertsContent = step.slice.size && step.slice.content.firstChild ? true : false
-    console.log('stepDeletesContent', stepDeletesContent)
-    console.log('stepInsertsContent', stepInsertsContent)
 
     for (let g = 0; g < tr.steps.length; g++) {
       // skipping if it's the same step or already paired
       if (g === i || matched.includes(g)) {
         continue
       }
       const peerStep = tr.steps[g] as ReplaceStep
+      if (!peerStep.slice) {
```

Comment:
please provide more extensive commentary in the code regarding why we need to skip empty peer steps here